spesifikasjoner/afpant/afpant-kjøperspantedokument/afpant-kjøperspantedokument-1-0-0.md

```diff
@@ -24,8 +24,11 @@ Wildcard "&ast;" kan erstattes med en vilkårlig streng (må være et gyldig fil
 
 ## Krav til dokumentreferanse
 - Benyttet dokumentreferanse må være unik i "signedmortgagedeed_&ast;.[sdo|xml]"
-Referansen må lages på en slik måte at den også er unik på tvers av banker. Dette ettersom megler skal lage en skjøtepakke som kan inneholde flere dokumenter fra flere kilder og dokumentreferansene i en pakke må være unike. 
+Referansen må lages på en slik måte at den også er unik på tvers av banker. Dette ettersom megler skal lage en skjøtepakke som kan inneholde flere dokumenter fra flere kilder og dokumentreferansene i en pakke må være unike. Id må være unikt på tvers av saker i den enkelte bank og på tvers av alle banker. Bruk av bankId er derfor ønskelig.
```

Comment:
nitpick: BankId kan forstås som noe annet enn det vi mener her vel? "intern saksId hos bank"? "bankens interne Id"? "lånesakId"?

Reply:
Nei, jeg tenkte egentlig på bankid her. Som i den id'en alle banker har.. Den som man ser i f.eks først i kontonummeret osv. Tanken er at en unik id i en bank kan gå igjen i en annen bank, ved å inkludere bankid så har man skilt det fra de andre bankene. Hos oss er f.eks caseid bare en db sekvens, den kan dermed være lik for flere banker. 
Formatet kan da som ett eksempel være det jeg skrev 
 [caseId]-[dokumentId]-[bankId] "12345-1234567899057" eller rett og slett en UUID, den vil jo være unik og trenger ikke noe bankid . I eksemplet over har jeg en caseid, og en dokumentID, denne inneholder tilfeldigvis bankid på slutten (9057) så jeg har derfor ikke -9057 etter dette igjen....

Reply:
Da forstår jeg - men begrepet "bankid" kan også (mis)forstås som "Vipps BankID" (som er noe helt annet - identitet og signering). Det var derfor jeg foreslo å bruke et annet begrep - har denne identifikatoren i kontonummer et annet navn enn "bankid", eller er det gjengs begrep?

Reply:
Bankid er nok et ukjent begrep. Bankregisternummer er nok et mer kjent begrep for de fire første sifrene i et kontonummer.

Reply:
tja, ser at visma kaller det "Bankens registernummer" https://www.visma.no/eaccounting/regnskapsordbok/k/kontonummer/
vet ikke hva som er det "riktige" 
BankID nå er vel ratt det man sier om bankid pålogging osv.. men i brukte bankid lenge før bankid kom, endre gjerne til noe som er mer riktig hvis noen har det offisielle begrepet..

Reply:
Høres bra ut - da blir det jo en ganske unik ID på tvers av alle banker om man bruker Jon Egil sitt forslag (men kaller det "bankregisternummer" og ikke "bankid"), eks:
[caseId]-[dokumentId]-[bankregisternummer]

Reply:
endret til bankregisternummer